docs/topics/js/js-interop.md

```diff
@@ -210,4 +210,41 @@ function usingAsOperator(s) {
     var tmp$;
     return typeof (tmp$ = s) === 'string' ? tmp$ : throwCCE();
 }
-```
\ No newline at end of file
+```
+
+## Equality
+
+Kotlin/JS has particular semantics for equality comparison. 
```

Comment:
```suggestion
Kotlin/JS has particular semantics for equality comparison compared to other platforms. 
```

Reply:
Wrote it as "...for equality checks compared to other platforms" to avoid repeat comparison-compared one after the other. WDYT?